kinode/packages/app-store/app-store/src/lib.rs

```diff
@@ -131,6 +132,11 @@ fn handle_message(
                     response.send()?;
                 }
             }
+            Req::RemoteRequest(remote_request) => match remote_request {
+                RemoteRequest::Ping => {
+                    Response::new().body(RemoteResponse::Ping).send()?;
+                }
+            },
```

Comment:
Since the purpose of the ping is to determine whether or not a node is mirroring a given app, why don't we just go ahead and actually encode the requested package name into the request? @barraguda is there a reason why the app store doesn't have a *real* mirror-check remote request-response?